pkg/cloudprovider/vsphereparavirtual/instances.go

```diff
@@ -44,7 +44,9 @@ type instances struct {
 }
 
 const (
-	providerPrefix = ProviderName + "://"
+	// providerPrefix is the Kubernetes cloud provider prefix for this
+	// cloud provider.
+	providerPrefix = "vsphere://"
```

Comment:
does this need to match `ProviderName()`? if so and if a constant is too much, is it worth adding a comment for that?

Reply:
Sure